res/controllers/Vestax-VCI-100MKII-scripts.js

```diff
@@ -257,32 +257,53 @@ VCI102.next_effect = function(ch, midino, value, status, group) {
     }
 };
 
-VCI102.loopSize = [4, 4, 4, 4];  // saved beatloop_size
-
 VCI102.loop = function(ch, midino, value, status, group) {
     if (value) {
         if (engine.getValue(group, "loop_enabled")) {
             engine.setValue(group, "reloop_toggle", 1);
-            // restore beatloop_size
-            engine.setValue(group, "beatloop_size", VCI102.loopSize[ch]);
         } else {
-            VCI102.loopSize[ch] = engine.getValue(group, "beatloop_size");
             engine.setValue(group, "beatloop_activate", 1);
+            engine.setValue(group, "beatloop_activate", 0);
         }
     }
 };
 
 VCI102.reloop = function(ch, midino, value, status, group) {
     if (value) {
         if (engine.getValue(group, "loop_enabled")) {
+            engine.setValue(group, "loop_out", 1);
```

Comment:
I don't understand why this button is using loop_out. It is unrelated to reloop_toggle. reloop_toggle should be accessible when loops are disabled.